public/stylesheets/dark.scss

```diff
@@ -52,9 +52,15 @@ a {
     border-color: #3396ff !important;
 }
 
-input[type="text"] {

```

Comment:
As the comment on top of the page says can you please edit the version number in application.ejs when changing the stylesheets?